topics/single-cell/tutorials/pseudobulk-analysis/tutorial.md

```diff
@@ -111,9 +120,9 @@ The data object, which you will import from Zenodo into Galaxy via the provided
 
 In this step, our goal is to perform a "bioinformatic cell sorting" based on the annotated clusters of the single-cell data.
 
-To start a pseudobulk analysis, ensure that the AnnData object you will use contains all the necessary metadata for ***"pseudobulking"***. For example, key annotations, such as cell_type, condition, disease, and batch, should be present. Most importantly, the AnnData object has to include a layer with the raw counts of gene expression.
+To start a pseudobulk analysis, ensure that the `AnnData` object contains all the necessary metadata for ***pseudobulking***. This includes key annotations such as `annotated` or `cell_type`, `condition`, `disease`, and `batch`. These labels may vary depending on how the dataset was originally annotated.
 
```

Comment:
Add Anndata inspect step (with general, obs and var inpsection) so that the user can see what exactly annotated in the anndata.